### DISCUSSION TITLE
dunfell-l4t-v32.5.0 ./dosdcard.sh ?

### DISCUSSION THREAD

**Discussion by bkirkTag** [OE4T/meta-tegra#634] [Q&A] [answered]
Hi,
I'v been using on the meta-tegra @ dunfell-l4t-v32.4.3 layer for a while and been using for the sdcard-image generation script.
The script generates my image.sdcard file (including all partitions needed ). :) MACHINE = "jetson-nano-qspi-sd"
```
[   0.0705 ] Copying signed file in /media/bk/extraDisk/yoctoXXXXJetson/build/tmp/deploy/images/jetson-nano-qspi-sd/tmpSD/signed
Init...TBC...RP1...EBT...WB0...BPF...BPF-DTB...FX...TOS...DTB...LNX...EKS...BMP...RP4...APP...[OK]
Writing...APP...TBC...RP1...EBT...WB0...BPF...TOS...DTB...LNX...EKS...BMP...RP4...[OK: core-image-minimal.sdcard]

```

Now i'm trying to switch to the newer branch dunfell-l4t-v32.5.0 and i ALMOST got it work - except the sdcard-image generation , I only got one partition:
```
Copying signed file in /media/bk/extraDisk/yoctoXXXXJetson/build/tmp/deploy/images/jetson-nano-devkit/tmpSD/signed
Init...APP...[OK]
Writing...APP...[OK: core-image-minimal.sdcard
```
I downloaded the jetpack 4.5 sdk files and changed my local.conf  MACHINE = "jetson-nano-devkit"

1) Anybody seen same issue, can somebody tell me what can be the issue?
2) Isn't the ./dosdcard.sh still meant for generating the sdcard-image?

**Comment by madisongh** [comment, marked as answer]
With L4T R32.5.0, NVIDIA moved all of the boot partitions for the Nano into the SPI flash, so now there's only the APP partition on the SDcard.

If you're upgrading, make sure you also re-flash the via USB (you can use `./doflash.sh --spi-only` for this) so the SPI flash gets the bootloader bits.

**Reply by bkirkTag** [reply]
Thanks, it make sense then with just one partition on the SD-image. I will try to update the SPI flash with the boot partitions.
